app/models/activity.rb

```diff
@@ -325,10 +327,18 @@ def is_project?
   end
 
   def is_gcrf_funded?
-    parent.present? && associated_fund.roda_identifier_fragment == "GCRF"
+    parent.present? && associated_fund.is_gcrf_fund?
   end
 
   def is_newton_funded?
-    parent.present? && associated_fund.roda_identifier_fragment == "NF"
+    parent.present? && associated_fund.is_newton_fund?
+  end
+
+  def is_newton_fund?
```

Comment:
@pezholio I can see that this is useful in tests, however I can having this exposed in the `Activity` model risks folks using `is_newton_fund?` by mistake when they mean `is_newton_funded?`.

Reply:
Yeah, you're right. I'll try and think of some nicer wording.